KomaMRIPlots/src/ui/DisplayFunctions.jl

```diff
@@ -154,38 +154,38 @@ julia> plot_seq(seq)
 ```
 """
 function plot_seq(
-      seq::Sequence; 
-      width=nothing, 
-      height=nothing, 
+      seq::Sequence;
+      width=nothing,
+      height=nothing,
       slider=false,
-      show_seq_blocks=false, 
-      darkmode=false, 
-      max_rf_samples=Inf, 
-      range=[], 
-      title="", 
+      show_seq_blocks=false,
+      darkmode=false,
+      max_rf_samples=Inf,
+      range=[],
+      title="",
       xaxis="x",
-      yaxis="y", 
+      yaxis="y",
       showlegend=true
   )
 	idx = ["Gx" "Gy" "Gz"]
 	N = length(seq)
 	O = size(seq.RF,1)
-	ΔT = KomaMRICore.durs(seq)
+	ΔT = KomaMRIBase.durs(seq)
```

Comment:
Is this different from `seq.DUR`? Maybe we should replace these and delete `durs`. I think is just legacy code.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/bdc397d3822bcf78b6faa39c8182eaa62ba2618f